# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1207760256 14400
#      Wed Apr 09 12:57:36 2008 -0400
# Branch release-3-0-x
# Node ID eb0a86b3deb90812d41068ae358e06ba25524db2
# Parent  c7070f6cadc8d577f45a79cf00bfb65da33f25b4
don't embed newline in warning messages in deprecated functions

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,59 @@
+2008-04-09  John W. Eaton  <jwe@octave.org>
+
+	* deprecated/beta_cdf.m, deprecated/beta_inv.m,
+	deprecated/beta_pdf.m, deprecated/beta_rnd.m,
+	deprecated/binomial_cdf.m, deprecated/binomial_inv.m,
+	deprecated/binomial_pdf.m, deprecated/binomial_rnd.m,
+	deprecated/chisquare_cdf.m, deprecated/chisquare_inv.m,
+	deprecated/chisquare_pdf.m, deprecated/chisquare_rnd.m,
+	deprecated/clearplot.m, deprecated/clg.m, deprecated/com2str.m,
+	deprecated/exponential_cdf.m, deprecated/exponential_inv.m,
+	deprecated/exponential_pdf.m, deprecated/exponential_rnd.m,
+	deprecated/f_cdf.m, deprecated/f_inv.m, deprecated/f_pdf.m,
+	deprecated/f_rnd.m, deprecated/gamma_cdf.m,
+	deprecated/gamma_inv.m, deprecated/gamma_pdf.m,
+	deprecated/gamma_rnd.m, deprecated/geometric_cdf.m,
+	deprecated/geometric_inv.m, deprecated/geometric_pdf.m,
+	deprecated/geometric_rnd.m, deprecated/hypergeometric_cdf.m,
+	deprecated/hypergeometric_inv.m, deprecated/hypergeometric_pdf.m,
+	deprecated/hypergeometric_rnd.m, deprecated/intersection.m,
+	deprecated/is_bool.m, deprecated/is_complex.m,
+	deprecated/is_list.m, deprecated/is_matrix.m,
+	deprecated/is_scalar.m, deprecated/is_square.m,
+	deprecated/is_stream.m, deprecated/is_struct.m,
+	deprecated/is_symmetric.m, deprecated/is_vector.m,
+	deprecated/isstr.m, deprecated/lchol.m,
+	deprecated/lognormal_cdf.m, deprecated/lognormal_inv.m,
+	deprecated/lognormal_pdf.m, deprecated/lognormal_rnd.m,
+	deprecated/meshdom.m, deprecated/normal_cdf.m,
+	deprecated/normal_inv.m, deprecated/normal_pdf.m,
+	deprecated/normal_rnd.m, deprecated/pascal_cdf.m,
+	deprecated/pascal_inv.m, deprecated/pascal_pdf.m,
+	deprecated/pascal_rnd.m, deprecated/poisson_cdf.m,
+	deprecated/poisson_inv.m, deprecated/poisson_pdf.m,
+	deprecated/poisson_rnd.m, deprecated/polyinteg.m,
+	deprecated/setstr.m, deprecated/spatan2.m, deprecated/spchol.m,
+	deprecated/spchol2inv.m, deprecated/spcholinv.m,
+	deprecated/spcumprod.m, deprecated/spcumsum.m, deprecated/spdet.m,
+	deprecated/spdiag.m, deprecated/spfind.m, deprecated/spinv.m,
+	deprecated/spkron.m, deprecated/splchol.m, deprecated/splu.m,
+	deprecated/spmax.m, deprecated/spmin.m, deprecated/spprod.m,
+	deprecated/spqr.m, deprecated/spsum.m, deprecated/spsumsq.m,
+	deprecated/struct_contains.m, deprecated/struct_elements.m,
+	deprecated/t_cdf.m, deprecated/t_inv.m, deprecated/t_pdf.m,
+	deprecated/t_rnd.m, deprecated/uniform_cdf.m,
+	deprecated/uniform_inv.m, deprecated/uniform_pdf.m,
+	deprecated/uniform_rnd.m, deprecated/weibcdf.m,
+	deprecated/weibinv.m, deprecated/weibpdf.m, deprecated/weibrnd.m,
+	deprecated/weibull_cdf.m, deprecated/weibull_inv.m,
+	deprecated/weibull_pdf.m, deprecated/weibull_rnd.m,
+	deprecated/wiener_rnd.m: Don't embed newline in warning message.
+
 2008-04-09  David Bateman  <dbateman@free.fr>
 
 	* testfun/assert.m: Don't allow cond and expected to be lists.
 
 2008-04-04  David Bateman  <dbateman@free.fr>
 
 	* deprecated/beta_cdf.m deprecated/beta_inv.m
 	deprecated/beta_pdf.m deprecated/beta_rnd.m
diff --git a/scripts/deprecated/beta_cdf.m b/scripts/deprecated/beta_cdf.m
--- a/scripts/deprecated/beta_cdf.m
+++ b/scripts/deprecated/beta_cdf.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function cdf = beta_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["beta_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use betacdf instead"]);
+             "beta_cdf is obsolete and will be removed from a future version of Octave; please use betacdf instead");
   endif
 
  cdf =  betacdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/beta_inv.m b/scripts/deprecated/beta_inv.m
--- a/scripts/deprecated/beta_inv.m
+++ b/scripts/deprecated/beta_inv.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function inv = beta_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["beta_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use betainv instead"]);
+             "beta_inv is obsolete and will be removed from a future version of Octave; please use betainv instead");
   endif
 
  inv =  betainv (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/beta_pdf.m b/scripts/deprecated/beta_pdf.m
--- a/scripts/deprecated/beta_pdf.m
+++ b/scripts/deprecated/beta_pdf.m
@@ -28,15 +28,14 @@
 ## Deprecated in version 3.0
 
 function pdf = beta_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["beta_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use betapdf instead"]);
+             "beta_pdf is obsolete and will be removed from a future version of Octave; please use betapdf instead");
   endif
 
  pdf =  betapdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/beta_rnd.m b/scripts/deprecated/beta_rnd.m
--- a/scripts/deprecated/beta_rnd.m
+++ b/scripts/deprecated/beta_rnd.m
@@ -34,15 +34,14 @@
 ## Deprecated in version 3.0
 
 function rnd = beta_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["beta_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use betarnd instead"]);
+             "beta_rnd is obsolete and will be removed from a future version of Octave; please use betarnd instead");
   endif
 
  rnd =  betarnd (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/binomial_cdf.m b/scripts/deprecated/binomial_cdf.m
--- a/scripts/deprecated/binomial_cdf.m
+++ b/scripts/deprecated/binomial_cdf.m
@@ -28,15 +28,14 @@
 ## Deprecated in version 3.0
 
 function cdf = binomial_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["binomial_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use binocdf instead"]);
+             "binomial_cdf is obsolete and will be removed from a future version of Octave; please use binocdf instead");
   endif
 
  cdf =  binocdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/binomial_inv.m b/scripts/deprecated/binomial_inv.m
--- a/scripts/deprecated/binomial_inv.m
+++ b/scripts/deprecated/binomial_inv.m
@@ -28,15 +28,14 @@
 ## Deprecated in version 3.0
 
 function inv = binomial_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["binomial_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use binoinv instead"]);
+             "binomial_inv is obsolete and will be removed from a future version of Octave; please use binoinv instead");
   endif
 
  inv =  binoinv (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/binomial_pdf.m b/scripts/deprecated/binomial_pdf.m
--- a/scripts/deprecated/binomial_pdf.m
+++ b/scripts/deprecated/binomial_pdf.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function pdf = binomial_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["binomial_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use binopdf instead"]);
+             "binomial_pdf is obsolete and will be removed from a future version of Octave; please use binopdf instead");
   endif
 
  pdf =  binopdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/binomial_rnd.m b/scripts/deprecated/binomial_rnd.m
--- a/scripts/deprecated/binomial_rnd.m
+++ b/scripts/deprecated/binomial_rnd.m
@@ -34,15 +34,14 @@
 ## Deprecated in version 3.0
 
 function rnd = binomial_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["binomial_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use binornd instead"]);
+             "binomial_rnd is obsolete and will be removed from a future version of Octave; please use binornd instead");
   endif
 
  rnd =  binornd (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/chisquare_cdf.m b/scripts/deprecated/chisquare_cdf.m
--- a/scripts/deprecated/chisquare_cdf.m
+++ b/scripts/deprecated/chisquare_cdf.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function cdf = chisquare_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["chisquare_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use chi2cdf instead"]);
+             "chisquare_cdf is obsolete and will be removed from a future version of Octave; please use chi2cdf instead");
   endif
 
  cdf =  chi2cdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/chisquare_inv.m b/scripts/deprecated/chisquare_inv.m
--- a/scripts/deprecated/chisquare_inv.m
+++ b/scripts/deprecated/chisquare_inv.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function inv = chisquare_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["chisquare_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use chi2inv instead"]);
+             "chisquare_inv is obsolete and will be removed from a future version of Octave; please use chi2inv instead");
   endif
 
  inv =  chi2inv (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/chisquare_pdf.m b/scripts/deprecated/chisquare_pdf.m
--- a/scripts/deprecated/chisquare_pdf.m
+++ b/scripts/deprecated/chisquare_pdf.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function pdf = chisquare_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["chisquare_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use chi2pdf instead"]);
+             "chisquare_pdf is obsolete and will be removed from a future version of Octave; please use chi2pdf instead");
   endif
 
  pdf =  chi2pdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/chisquare_rnd.m b/scripts/deprecated/chisquare_rnd.m
--- a/scripts/deprecated/chisquare_rnd.m
+++ b/scripts/deprecated/chisquare_rnd.m
@@ -33,15 +33,14 @@
 ## Deprecated in version 3.0
 
 function rnd = chisquare_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["chisquare_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use chi2rnd instead"]);
+             "chisquare_rnd is obsolete and will be removed from a future version of Octave; please use chi2rnd instead");
   endif
 
  rnd =  chi2rnd (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/clearplot.m b/scripts/deprecated/clearplot.m
--- a/scripts/deprecated/clearplot.m
+++ b/scripts/deprecated/clearplot.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function clearplot ()
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["function clearplot () is obsolete and will be removed from a future\n",
-	       "version of Octave, please use clf instead"]);
+             "clearplot is obsolete and will be removed from a future version of Octave; please use clf instead");
   endif
 
   clf ();
 
 endfunction
diff --git a/scripts/deprecated/clg.m b/scripts/deprecated/clg.m
--- a/scripts/deprecated/clg.m
+++ b/scripts/deprecated/clg.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function clg ()
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["function clg () is obsolete and will be removed from a future\n",
-	       "version of Octave, please use clf instead"]);
+             "clg is obsolete and will be removed from a future version of Octave, please use clf instead");
   endif
 
   clf ();
 
 endfunction
diff --git a/scripts/deprecated/com2str.m b/scripts/deprecated/com2str.m
--- a/scripts/deprecated/com2str.m
+++ b/scripts/deprecated/com2str.m
@@ -36,18 +36,17 @@
 ## Deprecated in version 3.0
 
 function retval = com2str (zz, flg)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["com2str is obsolete and will be removed from a future\n",
-	       "version of Octave, please use num2str instead"]);
+             "com2str is obsolete and will be removed from a future version of Octave; please use num2str instead");
   endif
 
   if (nargin < 1 || nargin > 2)
     print_usage ();
   endif
   if (nargin == 1)
     flg = 0;
   endif
diff --git a/scripts/deprecated/exponential_cdf.m b/scripts/deprecated/exponential_cdf.m
--- a/scripts/deprecated/exponential_cdf.m
+++ b/scripts/deprecated/exponential_cdf.m
@@ -31,18 +31,17 @@
 ## Deprecated in version 3.0
 
 function cdf = exponential_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["exponential_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use expcdf instead"]);
+             "exponential_cdf is obsolete and will be removed from a future version of Octave; please use expcdf instead");
   endif
 
  if (nargin > 1)
    varargin{2} = 1 ./ varargin{2};
  endif
 
  cdf =  expcdf (varargin{:});
 
diff --git a/scripts/deprecated/exponential_inv.m b/scripts/deprecated/exponential_inv.m
--- a/scripts/deprecated/exponential_inv.m
+++ b/scripts/deprecated/exponential_inv.m
@@ -29,18 +29,17 @@
 ## Deprecated in version 3.0
 
 function inv = exponential_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["exponential_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use expinv instead"]);
+             "exponential_inv is obsolete and will be removed from a future version of Octave; please use expinv instead");
   endif
 
  if (nargin > 1)
    varargin{2} = 1 ./ varargin{2};
  endif
 
  inv =  expinv (varargin{:});
 
diff --git a/scripts/deprecated/exponential_pdf.m b/scripts/deprecated/exponential_pdf.m
--- a/scripts/deprecated/exponential_pdf.m
+++ b/scripts/deprecated/exponential_pdf.m
@@ -28,18 +28,17 @@
 ## Deprecated in version 3.0
 
 function pdf = exponential_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["exponential_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use exppdf instead"]);
+             "exponential_pdf is obsolete and will be removed from a future version of Octave; please use exppdf instead");
   endif
 
  if (nargin > 1)
    varargin{2} = 1 ./ varargin{2};
  endif
 
  pdf =  exppdf (varargin{:});
 
diff --git a/scripts/deprecated/exponential_rnd.m b/scripts/deprecated/exponential_rnd.m
--- a/scripts/deprecated/exponential_rnd.m
+++ b/scripts/deprecated/exponential_rnd.m
@@ -34,18 +34,17 @@
 ## Deprecated in version 3.0
 
 function rnd = exponential_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["exponential_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use exprnd instead"]);
+             "exponential_rnd is obsolete and will be removed from a future version of Octave; please use exprnd instead");
   endif
 
  if (nargin > 0)
    varargin{1} = 1 ./ varargin{1};
  endif
 
  rnd =  exprnd (varargin{:});
 
diff --git a/scripts/deprecated/f_cdf.m b/scripts/deprecated/f_cdf.m
--- a/scripts/deprecated/f_cdf.m
+++ b/scripts/deprecated/f_cdf.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function cdf = f_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["f_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use fcdf instead"]);
+             "f_cdf is obsolete and will be removed from a future version of Octave; please use fcdf instead");
   endif
 
  cdf =  fcdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/f_inv.m b/scripts/deprecated/f_inv.m
--- a/scripts/deprecated/f_inv.m
+++ b/scripts/deprecated/f_inv.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function inv = f_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["f_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use finv instead"]);
+             "f_inv is obsolete and will be removed from a future version of Octave; please use finv instead");
   endif
 
  inv =  finv (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/f_pdf.m b/scripts/deprecated/f_pdf.m
--- a/scripts/deprecated/f_pdf.m
+++ b/scripts/deprecated/f_pdf.m
@@ -29,15 +29,14 @@
 ## Deprecated in version 3.0
 
 function pdf = f_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["f_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use fpdf instead"]);
+             "f_pdf is obsolete and will be removed from a future version of Octave; please use fpdf instead");
   endif
 
  pdf =  fpdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/f_rnd.m b/scripts/deprecated/f_rnd.m
--- a/scripts/deprecated/f_rnd.m
+++ b/scripts/deprecated/f_rnd.m
@@ -35,15 +35,14 @@
 ## Deprecated in version 3.0
 
 function rnd = f_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["f_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use frnd instead"]);
+             "f_rnd is obsolete and will be removed from a future version of Octave; please use frnd instead");
   endif
 
  rnd =  frnd (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/gamma_cdf.m b/scripts/deprecated/gamma_cdf.m
--- a/scripts/deprecated/gamma_cdf.m
+++ b/scripts/deprecated/gamma_cdf.m
@@ -29,18 +29,17 @@
 ## Deprecated in version 3.0
 
 function cdf = gamma_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["gamma_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use gamcdf instead"]);
+             "gamma_cdf is obsolete and will be removed from a future version of Octave; please use gamcdf instead");
   endif
 
  if (nargin > 2)
    varargin{3} = 1 ./ varargin{3};
  endif
 
  cdf =  gamcdf (varargin{:});
 
diff --git a/scripts/deprecated/gamma_inv.m b/scripts/deprecated/gamma_inv.m
--- a/scripts/deprecated/gamma_inv.m
+++ b/scripts/deprecated/gamma_inv.m
@@ -29,18 +29,17 @@
 ## Deprecated in version 3.0
 
 function inv = gamma_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["gamma_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use gaminv instead"]);
+             "gamma_inv is obsolete and will be removed from a future version of Octave; please use gaminv instead");
   endif
 
  if (nargin > 2)
    varargin{3} = 1 ./ varargin{3};
  endif
 
  inv =  gaminv (varargin{:});
 
diff --git a/scripts/deprecated/gamma_pdf.m b/scripts/deprecated/gamma_pdf.m
--- a/scripts/deprecated/gamma_pdf.m
+++ b/scripts/deprecated/gamma_pdf.m
@@ -29,18 +29,17 @@
 ## Deprecated in version 3.0
 
 function pdf = gamma_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["gamma_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use gampdf instead"]);
+             "gamma_pdf is obsolete and will be removed from a future version of Octave; please use gampdf instead");
   endif
 
  if (nargin > 2)
    varargin{3} = 1 ./ varargin{3};
  endif
 
  pdf =  gampdf (varargin{:});
 
diff --git a/scripts/deprecated/gamma_rnd.m b/scripts/deprecated/gamma_rnd.m
--- a/scripts/deprecated/gamma_rnd.m
+++ b/scripts/deprecated/gamma_rnd.m
@@ -34,18 +34,17 @@
 ## Deprecated in version 3.0
 
 function rnd = gamma_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["gamma_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use gamrnd instead"]);
+             "gamma_rnd is obsolete and will be removed from a future version of Octave; please use gamrnd instead");
   endif
 
  if (nargin > 1)
    varargin{2} = 1 ./ varargin{2};
  endif
 
  rnd =  gamrnd (varargin{:});
 
diff --git a/scripts/deprecated/geometric_cdf.m b/scripts/deprecated/geometric_cdf.m
--- a/scripts/deprecated/geometric_cdf.m
+++ b/scripts/deprecated/geometric_cdf.m
@@ -28,15 +28,14 @@
 ## Deprecated in version 3.0
 
 function cdf = geometric_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["geometric_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use geocdf instead"]);
+             "geometric_cdf is obsolete and will be removed from a future version of Octave; please use geocdf instead");
   endif
 
  cdf =  geocdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/geometric_inv.m b/scripts/deprecated/geometric_inv.m
--- a/scripts/deprecated/geometric_inv.m
+++ b/scripts/deprecated/geometric_inv.m
@@ -28,15 +28,14 @@
 ## Deprecated in version 3.0
 
 function inv = geometric_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["geometric_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use geoinv instead"]);
+             "geometric_inv is obsolete and will be removed from a future version of Octave; please use geoinv instead");
   endif
 
  inv =  geoinv (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/geometric_pdf.m b/scripts/deprecated/geometric_pdf.m
--- a/scripts/deprecated/geometric_pdf.m
+++ b/scripts/deprecated/geometric_pdf.m
@@ -28,15 +28,14 @@
 ## Deprecated in version 3.0
 
 function pdf = geometric_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["geometric_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use geopdf instead"]);
+             "geometric_pdf is obsolete and will be removed from a future version of Octave; please use geopdf instead");
   endif
 
  pdf =  geopdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/geometric_rnd.m b/scripts/deprecated/geometric_rnd.m
--- a/scripts/deprecated/geometric_rnd.m
+++ b/scripts/deprecated/geometric_rnd.m
@@ -34,15 +34,14 @@
 ## Deprecated in version 3.0
 
 function rnd = geometric_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["geometric_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use geornd instead"]);
+             "geometric_rnd is obsolete and will be removed from a future version of Octave; please use geornd instead");
   endif
 
  rnd =  geornd (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/hypergeometric_cdf.m b/scripts/deprecated/hypergeometric_cdf.m
--- a/scripts/deprecated/hypergeometric_cdf.m
+++ b/scripts/deprecated/hypergeometric_cdf.m
@@ -35,17 +35,16 @@
 ## Deprecated in version 3.0
 
 function cdf = hypergeometric_cdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["hypergeometric_cdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use hygecdf instead"]);
+             "hypergeometric_cdf is obsolete and will be removed from a future version of Octave; please use hygecdf instead");
   endif
 
  cdf =  hygecdf (varargin{:});
 
 endfunction
 
 
diff --git a/scripts/deprecated/hypergeometric_inv.m b/scripts/deprecated/hypergeometric_inv.m
--- a/scripts/deprecated/hypergeometric_inv.m
+++ b/scripts/deprecated/hypergeometric_inv.m
@@ -32,15 +32,14 @@
 ## Deprecated in version 3.0
 
 function inv = hypergeometric_inv (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["hypergeometric_inv is obsolete and will be removed from a future\n",
-	       "version of Octave, please use hygeinv instead"]);
+             "hypergeometric_inv is obsolete and will be removed from a future version of Octave; please use hygeinv instead");
   endif
 
  inv =  hygeinv (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/hypergeometric_pdf.m b/scripts/deprecated/hypergeometric_pdf.m
--- a/scripts/deprecated/hypergeometric_pdf.m
+++ b/scripts/deprecated/hypergeometric_pdf.m
@@ -33,15 +33,14 @@
 ## Deprecated in version 3.0
 
 function pdf = hypergeometric_pdf (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["hypergeometric_pdf is obsolete and will be removed from a future\n",
-	       "version of Octave, please use hygepdf instead"]);
+             "hypergeometric_pdf is obsolete and will be removed from a future version of Octave; please use hygepdf instead");
   endif
 
  pdf =  hygepdf (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/hypergeometric_rnd.m b/scripts/deprecated/hypergeometric_rnd.m
--- a/scripts/deprecated/hypergeometric_rnd.m
+++ b/scripts/deprecated/hypergeometric_rnd.m
@@ -30,15 +30,14 @@
 ## Deprecated in version 3.0
 
 function rnd = hypergeometric_rnd (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["hypergeometric_rnd is obsolete and will be removed from a future\n",
-	       "version of Octave, please use hygernd instead"]);
+             "hypergeometric_rnd is obsolete and will be removed from a future version of Octave; please use hygernd instead");
   endif
 
   rnd = hygernd (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/intersection.m b/scripts/deprecated/intersection.m
--- a/scripts/deprecated/intersection.m
+++ b/scripts/deprecated/intersection.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function y = intersection (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["intersection is obsolete and will be removed from a future\n",
-	       "version of Octave, please use intersect instead"]);
+             "intersection is obsolete and will be removed from a future version of Octave; please use intersect instead");
   endif
 
   y = intersect (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_bool.m b/scripts/deprecated/is_bool.m
--- a/scripts/deprecated/is_bool.m
+++ b/scripts/deprecated/is_bool.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_bool (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_bool is obsolete and will be removed from a future\n",
-	       "version of Octave, please use isbool instead"]);
+             "is_bool is obsolete and will be removed from a future version of Octave; please use isbool instead");
   endif
 
   retval = isbool (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_complex.m b/scripts/deprecated/is_complex.m
--- a/scripts/deprecated/is_complex.m
+++ b/scripts/deprecated/is_complex.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_complex (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_complex is obsolete and will be removed from a future\n",
-	       "version of Octave, please use iscomplex instead"]);
+             "is_complex is obsolete and will be removed from a future version of Octave; please use iscomplex instead");
   endif
 
   retval = iscomplex (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_list.m b/scripts/deprecated/is_list.m
--- a/scripts/deprecated/is_list.m
+++ b/scripts/deprecated/is_list.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_list (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_list is obsolete and will be removed from a future\n",
-	       "version of Octave, please use islist instead"]);
+             "is_list is obsolete and will be removed from a future version of Octave; please use islist instead");
   endif
 
   retval = islist (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_matrix.m b/scripts/deprecated/is_matrix.m
--- a/scripts/deprecated/is_matrix.m
+++ b/scripts/deprecated/is_matrix.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_matrix (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_matrix is obsolete and will be removed from a future\n",
-	       "version of Octave, please use ismatrix instead"]);
+             "is_matrix is obsolete and will be removed from a future version of Octave; please use ismatrix instead");
   endif
 
   retval = ismatrix (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_scalar.m b/scripts/deprecated/is_scalar.m
--- a/scripts/deprecated/is_scalar.m
+++ b/scripts/deprecated/is_scalar.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_scalar (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_scalar is obsolete and will be removed from a future\n",
-	       "version of Octave, please use isscalar instead"]);
+             "is_scalar is obsolete and will be removed from a future version of Octave; please use isscalar instead");
   endif
 
   retval = isscalar (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_square.m b/scripts/deprecated/is_square.m
--- a/scripts/deprecated/is_square.m
+++ b/scripts/deprecated/is_square.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_square (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_square is obsolete and will be removed from a future\n",
-	       "version of Octave, please use issquare instead"]);
+             "is_square is obsolete and will be removed from a future version of Octave; please use issquare instead");
   endif
 
   retval = issquare (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_stream.m b/scripts/deprecated/is_stream.m
--- a/scripts/deprecated/is_stream.m
+++ b/scripts/deprecated/is_stream.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_stream (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_stream is obsolete and will be removed from a future\n",
-	       "version of Octave, please use isstream instead"]);
+             "is_stream is obsolete and will be removed from a future version of Octave; please use isstream instead");
   endif
 
   retval = isstream(varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_struct.m b/scripts/deprecated/is_struct.m
--- a/scripts/deprecated/is_struct.m
+++ b/scripts/deprecated/is_struct.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_struct (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_struct is obsolete and will be removed from a future\n",
-	       "version of Octave, please use isstruct instead"]);
+             "is_struct is obsolete and will be removed from a future version of Octave; please use isstruct instead");
   endif
 
   retval = isstruct (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_symmetric.m b/scripts/deprecated/is_symmetric.m
--- a/scripts/deprecated/is_symmetric.m
+++ b/scripts/deprecated/is_symmetric.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_symmetric (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_symmetric is obsolete and will be removed from a future\n",
-	       "version of Octave, please use issymmetric instead"]);
+             "is_symmetric is obsolete and will be removed from a future version of Octave; please use issymmetric instead");
   endif
 
   retval = issymmetric (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/is_vector.m b/scripts/deprecated/is_vector.m
--- a/scripts/deprecated/is_vector.m
+++ b/scripts/deprecated/is_vector.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = is_vector (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["is_vector is obsolete and will be removed from a future\n",
-	       "version of Octave, please use isvector instead"]);
+             "is_vector is obsolete and will be removed from a future version of Octave; please use isvector instead");
   endif
 
   retval = isvector (varargin{:});
 
 endfunction
diff --git a/scripts/deprecated/isstr.m b/scripts/deprecated/isstr.m
--- a/scripts/deprecated/isstr.m
+++ b/scripts/deprecated/isstr.m
@@ -26,15 +26,14 @@
 ## Deprecated in version 3.0
 
 function retval = isstr (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
-        ["isstr is obsolete and will be removed from a future\n",
-	       "version of Octave, please use ischar instead"]);
+             "isstr is obsolete and will be removed from a future version of Octave, please use ischar instead");
   endif
 
   retval = ischar (varargin{:});
 
 endfunction
